package.json

```diff
@@ -6,11 +6,11 @@
   "homepage": "https://github.com/okta/okta-auth-js",
   "license": "Apache-2.0",
   "type": "commonjs",
-  "main": "build/cjs/server/index.js",
-  "module": "build/lib/server/index.js",
+  "main": "build/cjs/index.js",
+  "module": "build/lib/index.js",
   "browser": "build/dist/okta-auth-js.umd.js",
   "react-native": "build/lib/server/index.js",
```

Comment:
remove? looks like it should be as same as `main`

Reply:
This is still providing a commonJS entry for those who need it. Maybe we can remove commonJS in this release? I'm trying to keep this PR as narrow as possible in focus, because it is so large.

Reply:
I mean the `react-native` entry. Looks like `server/index.js` has been removed from this PR, and the react-native should be able to consume the main field, as it points to the cjs bundle.

Reply:
react-native should be able to use either the CJS (main) or the ESM (module) entry point. Other than the bundling it is the same codebase which should work fine without a `window`.